src/rgw/support/rgw_support_bundle

```diff
@@ -83,6 +84,13 @@ class RGWSupportBundle:
                 shutil.copytree(CRASHDUMP_DIR,\
                     os.path.join(RGWSupportBundle._tmp_src, 'crash'))
 
+        # copy motr trace log files, if stacktrace=True
+        if stacktrace:
+            motr_trace_dir = os.path.join(log_dir, 'm0_client')
+            if os.path.exists(motr_trace_dir):
+                shutil.copytree(motr_trace_dir,\
```

Comment:
whats the compressed size of each m0trace file, ideally 16M one file expected, but if cleanup not active, old m0traces would be seen.
If compressed size around 2MB, then we can plan to include max last 5 m0traces files, to keep RGW support bundle in check.
@yeshpal-jain-seagate @papan-singh please raise if this approach would be an issue !?

Reply:
Added a logic in the rgw_support_bundle script, to fetch the latest 5 motr trace log files.

> [root@ssc-vm-g4-rhev4-0860 rgw]# tar -xvf rgw_SBag9s2g01_e884506cadb150157e923ad8e4873ac4_rgw.tar.gz
tar: Removing leading `/' from member names
/
rgw-1.log-20220306.gz
rgw_startup.log
rgw_startup.log-20220306.gz
rgw-1.log-20220227.gz
rgw_setup.log-20220304.gz
rgw_setup.log-20220225.gz
cortx-rpms
rgw_setup.log
**motr_trace_files/
motr_trace_files/m0trace.1822941
motr_trace_files/m0trace.1823015
motr_trace_files/m0trace.1838864
motr_trace_files/m0trace.1822853
motr_trace_files/m0trace.1822808**
rgw_setup.log-20220306.gz
rgw-1.log
cortx_rgw.conf